### PR TITLE
New partial json response filter

### PR DESCRIPTION
**Pull request by lukasnovy**
* Add info about new partial response filter
* Add info about scopes

### CODE REVIEW

**Review by LukasOsmik — APPROVED** [review]

File: apiary.apib [hunk 1]
```diff
@@ -415,6 +428,9 @@ Get loan detail.
 
 Investments of the loan selected by ID.
 
+**Required scopes**
+* none
```
Comment (LukasOsmik) [inline apiary.apib]: tady pro ten enpoint je asi vyzadovany scope viz radek 472, kde si ho pridaval pro tu GET metodu

Comment (lukasnovy) [reply apiary.apib]: Diky overim

**Review by tomix26 — APPROVED** [review]

File: apiary.apib [hunk 2]
```diff
@@ -240,14 +245,19 @@ and details in the response body:
...
+- ```GET https://api.zonky.cz/loans/marketplace?fields=id,url```
+- ```GET https://api.zonky.cz/loans/marketplace?fields=id,url,borrowerRelatedInfo.*```
+- ```GET https://api.zonky.cz/loans/marketplace?fields=id,photos.url```
+- ```GET https://api.zonky.cz/loans/marketplace?fields=id,photos.*```
 
-Example:
-Request: ```GET http://api.zonky/loans/marketplace?fields=id,url```
+Request:
+
+```GET https://api.zonky.cz/loans/marketplace?fields=id,url```
```
Comment (tomix26) [inline apiary.apib:260]: Could you please add an example of the filtering with `.` notation?

Comment (lukasnovy) [reply apiary.apib:260]: You mean response example? Because request examples are in previous section

Comment (tomix26) [reply apiary.apib:260]: You're right. So it's ok, sorry.

**Pull request merged**